toolchain/check/testdata/if_expr/fail_not_in_function.carbon

```diff
@@ -154,7 +154,7 @@ fn F() {
 // CHECK:STDOUT: }
 // CHECK:STDOUT:
 // CHECK:STDOUT: class @B {
-// CHECK:STDOUT:   %C.ref: %C.type = name_ref C, <unexpected>.inst21.loc4_24 [concrete = constants.%C.generic]
+// CHECK:STDOUT:   %C.ref: %C.type = name_ref C, <unexpected>.inst23.loc4_24 [concrete = constants.%C.generic]
```

Comment:
There's still some `<unexpected>` left, but in a very different context than the specifics one